config/deploy.rb

```diff
@@ -1,63 +1,69 @@
-require 'bundler/capistrano'
-require 'capistrano_colors'
-require 'dotenv/deployment/capistrano'
+# config valid for current version and patch releases of Capistrano
+lock '~> 3.11.0'
 
-set :default_environment, 'PATH' => '/opt/ruby-2.5.1/bin/:$PATH'
```

Comment:
I would recommend to keep it. The motivation is supporting different Ruby versions at the same time. Now it is used last-ruby only

Reply:
I see, we have it below